src/components/SVGViewer/SVGViewer.tsx

```diff
@@ -49,6 +49,8 @@ export interface SvgViewerProps {
   handleItemClick?: (metadataNode: HTMLElement) => void;
   // Error document load callback
   handleDocumentLoadError?: (error: Error) => void;
+  // Subscribe to SVGVieweSearch changes
+  handleSearchChange?: () => void;
```

Comment:
Would onSearchChange be a more suitable name?
Im kind of used to this convention:
Should we call the function onSearchChanged to indicate that this callback will be called when an event happen?

```
const Child = () => <button onChange={onChange} />
const Parent = () => {
const handleChange = {
setState();
}
return (
<Child onChange={handleChange} /> 
);
}
```

Reply:
Agree, but this is not the convention used in gearbox, so I just stick to the one there. We can update it later if it makes sense